src/test/scala/apps/cameraPipeCheck.scala

```diff
@@ -141,14 +189,10 @@ int main(int argc, char** argv) {
 
   test("deinterleave passes checks") {
     val typed = printTime(infer(nFun(h => nFun(w =>
-      deinterleave(h)(w) >> mapSeq(mapSeq(mapSeq(fun(x => x))))
+      deinterleave(h)(w)  // >> mapSeq(mapSeq(mapSeq(fun(x => x))))
```

Comment:
I believe that elevate is now inserting exactly that, maybe just remove the comment entirely?